src/logger.ts

```diff
@@ -59,6 +59,10 @@ class Logger {
     return this.log('info', msg);
   }
 
+  infoServer(data: any): Logger {
```

Comment:
It is not necessary to have this method in logger class. If you want to print for specific pattern like server response, please have the common method in api.ts

Reply:
Will change in the next commit.